src/domain/flow-system/services/src/flow/flow_executor_impl.rs

```diff
@@ -66,6 +67,12 @@ pub struct FlowExecutorImpl {
     ],
     durability: MessageConsumptionDurability::Durable,
 })]
+#[interface(dyn InitOnStartup)]
+#[meta(InitOnStartupMeta {
+    job_name: JOB_KAMU_FLOW_EXECUTOR_RECOVERY,
+    depends_on: &[],
```

Comment:
Q: Do we need to make a strict initialization order for `FlowExecutor`?

`OutboxExecutor -> TaskExecutor -> FlowExecutor`

Reply:
It's important the main loops don't start before initialization end, but there is no dependency between the initialization phases themselves.